custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml

```diff
@@ -260,3 +286,34 @@ card_light:
             border-radius: 14px;
             box-shadow: none;
           }
+    item3:
+      card:
+        type: "horizontal-stack"
```

Comment:
Breaks the dashboard if `horizontal-stack` is not used somewhere else in the dashboard.
Fix is to use the internal templates for 3 items on a row:
`list_3_items`